<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Face</class>
 <widget class="QDialog" name="Face">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QToolButton" name="toolButton_5">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resource/images/face/05.png</normaloff>:/resource/images/face/05.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="toolButton_2">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resource/images/face/02.png</normaloff>:/resource/images/face/02.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="toolButton_7">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resource/images/face/07.png</normaloff>:/resource/images/face/07.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="toolButton_1">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resource/images/face/01.png</normaloff>:/resource/images/face/01.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="toolButton_4">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resource/images/face/04.png</normaloff>:/resource/images/face/04.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="toolButton_8">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resource/images/face/08.png</normaloff>:/resource/images/face/08.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="toolButton_3">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resource/images/face/03.png</normaloff>:/resource/images/face/03.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="toolButton_6">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resource/images/face/06.png</normaloff>:/resource/images/face/06.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="toolButton_9">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/resource/images/face/09.png</normaloff>:/resource/images/face/09.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
